lib/element/command.js

```diff
@@ -193,14 +193,15 @@ class ElementCommand extends EventEmitter {
     try {
       if (WebdriverElementId) {
         return {
-          value: WebdriverElementId,
+          value: this.transport.toElement(WebdriverElementId),
           status: 0,
           result: {}
         };
       }
 
       if (this.element.webElement) {
```

Comment:
do we need to make this change? these are elements retrieved using commands from the `selenium-webdriver` library (see [using with selenium-webdriver](https://nightwatchjs.org/guide/extending-nightwatch/using-with-selenium-webdriver.html). I don't think we should to alter these elements.

Reply:
I'm not changing the webdriver element itself.
The `convertWebElement` returns the same "response" as the `locator.js` would do. 
I need to use
```js
webElement.value = this.transport.toElement(webElement.value);
```
to generate the same `value` (i.e. `{ 'element-6066...': '...' }`) as the `locator.js` returns.
If I don't do this the selenium tests in `test/src/runner/testRunnerWithSeleniumWebdriver.js` would fail.

Reply:
Nevermind, this is already a converted element. I'll move the convertWebElement to the transport to avoid changing the value.